app/server/utils/ussd/kenya_ussd_processor.py

```diff
@@ -76,5 +76,19 @@ def replace_vars(menu: UssdMenu, ussd_session: UssdSession, display_text: str, u
                     replacements.append(['%remaining_attempts%', "You have {} attempts remaining.".format(3 - user.pin_failed_attempts)])
             else:
                 replacements.append(['%remaining_attempts%', ''])
+        elif 'directory_listing' == menu.name or 'send_token_reason' == menu.name:
+            usages = user.get_most_relevant_transfer_usage()
+            menu_options = ''
+            for i, usage in enumerate(usages):
+                business_usage_string = None
+                if usage.translations is not None and user.preferred_language is not None:
+                    business_usage_string = usage.translations.get(
+                        user.preferred_language)
+                    print(business_usage_string)
+                if business_usage_string is None:
+                    business_usage_string = usage.name
+                message_option = '\n%d. %s' % (i+1, business_usage_string)
```

Comment:
How do new lines and tabs get translated to an sms? As a response in postman I just got everything back with spaces and no new lines even also for the other menu's

Reply:
ah... i actually haven't tested this. that is good to know. we should test it with the africas talking sandbox. definitely need to figure out how to fix it if it's not actually new lines

Reply:
If I want to do test it with africas talking sandbox where should I get started?

Reply:
sign up here:
https://account.africastalking.com/

go to sandbox, and then ussd, create an endpoint to an ngrok (https://ngrok.com/) server running on ur local machine, and then launch the africa's talking simulator :)

Reply:
Next lines do show up in the africa is talking sandbox:
I think it looks better with \n:
![image](https://user-images.githubusercontent.com/34537157/69254708-f45e5e00-0bb6-11ea-8fac-659591cf5984.png) vs
![image](https://user-images.githubusercontent.com/34537157/69254795-1526b380-0bb7-11ea-9b50-6a74cefcb6ac.png)
Should I:

1. Remove the \n from my code to make the menu's uniform
2. Or should we add \n to the other menu's (create a trello task for it)?

Reply:
Definitely agree that \n. looks better. My vote is number 2!

Reply:
see my PR for getting it working with sandbox. it fixes this issue